hera_cal/reflections.py

```diff
@@ -579,17 +582,24 @@ def sv_decomp(self, dfft, wgts=None, flags=None, keys=None, Nkeep=None,
 
     def project_svd_modes(self, dfft, umodes=None, svals=None, vmodes=None):
         """
-        Given two of the 3 SVD output vectors, project the input dfft data onto
-        the last remaining SVD vector.
+        Given two of the 3 SVD output matrices, project them onto the input dfft data
+        to estimate the last remaining SVD matrix. Note U and V are unitary matrices.
+
+        If
+            D = U S V
```

Comment:
typically, SVD is written as D = U S V^dagger

Reply:
yeah but for the purposes of the explanation in the doc string it's easier to express it this way, and in practice it makes no difference